README.md

```diff
@@ -3,9 +3,10 @@
 [![Build Status](https://travis-ci.org/cfpb/cfgov-refresh.png?branch=flapjack)](https://travis-ci.org/cfpb/cfgov-refresh?branch=flapjack)
 [![Code Climate](https://codeclimate.com/github/cfpb/cfgov-refresh.png?branch=flapjack)](https://codeclimate.com/github/cfpb/cfgov-refresh?branch=flapjack)
 
-The in-progress redesign of the [consumerfinance.gov](http://consumerfinance.gov) website.
+The redesign of the [www.consumerfinance.gov](http://www.consumerfinance.gov) website.
```

Comment:
As long as the code base we're talking about is just the redesigned portion of the site and not the full site, I think the distinction is a good one. It seems simpler for now to stick with "the redesign of..." rather than trying to work out how to say it's the current deployment of the redesigned portion of www.consumerfinance.gov.